### PR TITLE
feat (workload): [monitoring] windows dcr for text log collection

### PR DESCRIPTION
**Pull request by ferantivero**
actual merge via https://github.com/mspnp/iaas-baseline/pull/25

### CODE REVIEW

File: infra-as-code/bicep/monitoring.bicep [hunk 1]
```diff
@@ -71,6 +124,134 @@ resource dineLinuxDcrPolicyMonitoringContributorRoleAssignment 'Microsoft.Author
...
+      }
+      resourceType: {
+        value: windowsVmLogsCustomDataCollectionRule.type
+      }
+    }
+  }
+  dependsOn: []
+}
+
+@description('Ensure the managed identity for logs DCR DINE policies is has needed permissions.')
```
Comment (josevarela) [inline infra-as-code/bicep/monitoring.bicep]:
```suggestion
@description('Ensure the managed identity for logs DCR DINE policies has the required permissions.')
```

File: infra-as-code/bicep/monitoring.bicep [hunk 2]
```diff
@@ -71,6 +124,134 @@ resource dineLinuxDcrPolicyMonitoringContributorRoleAssignment 'Microsoft.Author
...
+  scope: resourceGroup()
+  name: guid(resourceGroup().id, logAnalyticsContributorRole.id, configureWindowsMachinesWithLogsDataCollectionRulePolicyAssignment.id)
+  properties: {
+    principalId: configureWindowsMachinesWithLogsDataCollectionRulePolicyAssignment.identity.principalId
+    roleDefinitionId: logAnalyticsContributorRole.id
+    principalType: 'ServicePrincipal'
+  }
+}
+
+@description('Ensure the managed identity for logs DCR DINE policies is has needed permissions.')
```
Comment (josevarela) [inline infra-as-code/bicep/monitoring.bicep]:
```suggestion
@description('Ensure the managed identity for logs DCR DINE policies has the required permissions.')
```

File: infra-as-code/bicep/networking.bicep [hunk 3]
```diff
@@ -22,53 +22,19 @@ targetScope = 'resourceGroup'
 @description('The spokes\'s regional affinity, must be the same as the hub\'s location. All resources tied to this spoke will also be homed in this region. The network team maintains this approved regional list which is a subset of zones with Availability Zone support.')
```
Comment (josevarela) [inline infra-as-code/bicep/networking.bicep]: references to the hub and spokes in the baseline seem out of place
```suggestion
@description('Region on which to create the VNnet. All resources tied to this VNnet will also be homed in this region. The region passed as a parameter is assumed to have Availability Zone support.')
```

**Review by josevarela — CHANGES_REQUESTED** [review]
Looks good overall. Including a couple of small suggestions in-line

File: infra-as-code/bicep/networking.bicep [hunk 4]
```diff
@@ -19,56 +19,22 @@ targetScope = 'resourceGroup'
   'japaneast'
   'southeastasia'
 ])
-@description('The spokes\'s regional affinity, must be the same as the hub\'s location. All resources tied to this spoke will also be homed in this region. The network team maintains this approved regional list which is a subset of zones with Availability Zone support.')
+@description('Region on which to create the VNnet. All resources tied to this VNnet will also be homed in this region. The region passed as a parameter is assumed to have Availability Zone support.')
```
Comment (josevarela) [inline infra-as-code/bicep/networking.bicep]:
```suggestion
@description('Region on which to create the VNet. All resources tied to this VNet will also be homed in this region. The region passed as a parameter is assumed to have Availability Zone support.')
```

**Review by josevarela — CHANGES_REQUESTED** [review]
Looks good. Small spelling correction suggested in-line.

**Comment by ferantivero** [comment]
actual chained long living merge via https://github.com/mspnp/iaas-baseline/pull/25

**Pull request merged**